presto-verifier/src/main/java/com/facebook/presto/verifier/PrestoVerifier.java

```diff
@@ -113,6 +116,9 @@ public int run(String[] args)
             queries = applyOverrides(config, queries);
             queries = filterQueryTypes(new SqlParser(getParserOptions()), config, queries);
             queries = filterQueries(queries);
+            if (config.getShadowWrites()) {
```

Comment:
I instead added a check that you have to have some form of writes enabled if this option is set